staging/src/k8s.io/apiserver/pkg/admission/configuration/validating_webhook_manager.go

```diff
@@ -29,13 +30,14 @@ import (
 	"k8s.io/client-go/informers"
 	admissionregistrationlisters "k8s.io/client-go/listers/admissionregistration/v1"
 	"k8s.io/client-go/tools/cache"
+	"k8s.io/client-go/tools/cache/synctrack"
 )
 
 // validatingWebhookConfigurationManager collects the validating webhook objects so that they can be called.
 type validatingWebhookConfigurationManager struct {
 	configuration *atomic.Value
 	lister        admissionregistrationlisters.ValidatingWebhookConfigurationLister
-	hasSynced     func() bool
+	delayer       *synctrack.Delayer
```

Comment:
Why did you make this a pointer?

Reply:
For consistency with the `configuration` member above, which also doesn't need to be a pointer...